dracut.spec

```diff
@@ -324,6 +324,7 @@ echo 'dracut_rescue_image="yes"' > $RPM_BUILD_ROOT%{dracutlibdir}/dracut.conf.d/
 %endif
 %{dracutlibdir}/modules.d/01systemd-ac-power
 %{dracutlibdir}/modules.d/01systemd-ask-password
+%{dracutlibdir}/modules.d/01systemd-timedated
```

Comment:
I thought it was alphabetically sorted :)

Reply:
Was is correct and now is again :smile: